eng/Versions.props

```diff
@@ -47,7 +47,7 @@
     <MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>3.7.0</MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>
     <MicrosoftCodeAnalysisVersionForTests>4.4.0-2.22416.9</MicrosoftCodeAnalysisVersionForTests>
     <DogfoodAnalyzersVersion>3.3.4-beta1.22418.3</DogfoodAnalyzersVersion>
-    <DogfoodNetAnalyzersVersion>7.0.0</DogfoodNetAnalyzersVersion>
+    <DogfoodNetAnalyzersVersion>8.0.0-preview1.22621.6</DogfoodNetAnalyzersVersion>
```

Comment:
So is 7.0.0 broken? If that's the case, I think it's important to fix that and release on NuGet?

Reply:
Yes, AnalysisLevel/AnalysisMode regression is first fixed in 7.0.101. We definitely need to release the corresponding NuGet package on NuGet.org. Tagging @jmarolf

Reply:
@mavasani The [`7.0.0` tag is pointing to e26a04c](https://github.com/dotnet/roslyn-analyzers/releases/tag/7.0.0), which is supposed to have the fix:

![image](https://user-images.githubusercontent.com/31348972/209084703-1962e309-f949-4841-a008-36a4aeb84adc.png)

Is the tag pointing to incorrect commit?

Reply:
I am guessing so